src/components/TextInput/BaseTextInput/index.tsx

```diff
@@ -447,6 +447,9 @@ function BaseTextInput(
                         styles.visibilityHidden,
                     ]}
                     onLayout={(e) => {
+                        if (e.nativeEvent.layout.width === 0 && e.nativeEvent.layout.height === 0) {
```

Comment:
This didn't work for me.
Please check this video for repro step

https://github.com/Expensify/App/pull/34787#issuecomment-1912913003

Reply:
What browser do you use? It works fine for me. Please check the recording attached above, I did the same step.

Reply:
ok works this time